config/systems.php

```diff
@@ -566,22 +567,22 @@
     ],
     100 => [
         'name' => 'Hubs',
-        'name_full' => null,
-        'name_short' => 'hubs',
+        'name_full' => 'Hubs',
+        'name_short' => 'Hubs',
         'manufacturer' => null,
         'active' => false,
     ],
     101 => [
         'name' => 'Events',
-        'name_full' => null,
+        'name_full' => 'Events',
         'name_short' => 'Events',
         'manufacturer' => null,
         'active' => true,
     ],
     102 => [
         'name' => 'Standalone',
         'name_full' => 'Standalone',
-        'name_short' => 'N/A',
+        'name_short' => 'Standalone',
```

Comment:
`name_short` is what appears in the Last N Games Played components on the user profile as well as the completion progress page. This is quite long - can we please revert this or change it to something else?

Reply:
`Standalone` is its shortest form yet represents its full name properly.
Presentation has to follow imo.
We have `Super Cassette Vision` as short name in there as well.
What do you suggest?

Reply:
Maybe `EXE` as that's what's on the icon?

Reply:
CV and SCV for (super) casette vision sound good to me, i'll adjust the config accordingly.
I'd favor EXE instead of N/A personally to make sure each system has an identifiable name. I had something different in mind for presentation when introducing short names, i can see how super short names are preferrable in the current uses though

Reply:
Sounds good, EXE seems like a better short name than N/A to me and I agree it's much more identifiable.

Reply:
updated to EXE, ECV (Epoch Casette Vision), and eSCV (Epoch Super Casette Vision) in https://github.com/RetroAchievements/RAWeb/pull/2172/commits/61bfa0c59c2ef772b337aea75a0f93a64eda9a99

Reply:
oh, `eSCV` is the emulator. i'll change it to `ESCV`.